test/intl402/RelativeTimeFormat/ctor-custom-get-prototype-poison-throws.js

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2019 Leo Balter. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-Intl.RelativeTimeFormat
+description: >
+  Return abrupt from Get Prototype from a custom NewTarget
+info: |
+  OrdinaryCreateFromConstructor ( constructor, intrinsicDefaultProto [ , internalSlotsList ] )
+
+  ...
+  2. Let proto be ? GetPrototypeFromConstructor(constructor, intrinsicDefaultProto).
+  ...
+
+  GetPrototypeFromConstructor ( constructor, intrinsicDefaultProto )
+
+  3. Let proto be ? Get(constructor, "prototype").
+  4. If Type(proto) is not Object, then
+    a. Let realm be ? GetFunctionRealm(constructor).
+    b. Set proto to realm's intrinsic object named intrinsicDefaultProto.
+  5. Return proto.
+features: [Intl.RelativeTimeFormat, Reflect, Proxy]
```

Comment:
since now you are adding tests of Intl.RelativeTimeFormat to this PR, please change your description to include it, or to remove these tests of Intl.RelativeTimeFormat from this PR and put into a different one. Either way is fine.